### DISCUSSION TITLE
Release plans for `3.4.0`

### DISCUSSION THREAD

**Discussion by atlj** [software-mansion/react-native-reanimated#4781] [Q&A] [answered]
Hey folks, there is a fix to an issue we had that's included with `3.4.0 nightly` versions and I was looking through the repository but couldn't find any release schedule or plans. I don't want to sound demanding, or push anything. Just wanted to know about the plans you have regarding the release of 3.4.0 so we can plan ahead. Have a nice day. Thanks!

**Comment by tjzel** [comment, marked as answer]
Hi @atlj, we plan to release it in the following week. We wanted to do it in this week but found many things that could be improved in the last second and they just piled up.

**Reply by atlj** [reply]
Thanks for the answer 🙏
